x-pack/test/functional/es_archives/fleet/agents/mappings.json

```diff
@@ -136,6 +136,9 @@
             "user_provided_metadata": {
               "type": "text"
             },
+            "current_error_events": {
```

Comment:
this name seems to be different than the other mapping? should it be `current_error_events` or `current_errors_events` - with `s` after errors?

for what it's worth, `current_error_events` reads better to me, and I am in strong favor of changing all instances of `current_errors_events` to `current_error_events`

Reply:
Going to do the change to `current_errors_events` everywhere 👍